app/views/pages/_editor_guide_text.html.erb

```diff
@@ -53,7 +53,7 @@
     <p>We support native
       <a href="https://shopify.github.io/liquid/" target="_blank" rel="noopener">Liquid tags</a> in our editor, but have created our own custom tags listed below:
     </p>
-    <h3><strong>dev.to Article/Post Embed</strong></h3>
+    <h3><strong><%= community_name %> Article/Post Embed</strong></h3>
     <p>All you need is the full link of the article:</p>
     <code>{% link https://dev.to/kazz/boost-your-productivity-using-markdown-1be %}</code>
```

Comment:
@rhymes or @lightalloy any idea on what we can do about these example links in the editor guide?

Reply:
the `link_tag` only works for internal articles, maybe we can change the ERB to output the full URL of that link?

Reply:
I meant that if we spin up another community, we'll need to make that link general and not dev specific?

Reply:
Right! Maybe we can have it point to the welcome thread :-)